ui/inspectors/system-section.reel/updates/updates.reel/updates.js

```diff
@@ -71,7 +71,12 @@ exports.Updates = Component.specialize(/** @lends Updates# */ {
 
     handleCheckDownloadAction: {
         value: function() {
-            this._updateService.checkAndDownload();
+            var self = this;
+            this._updateService.checkAndDownload().then(function() {
```

Comment:
As neither `chackAndDownload` or `getInfo` are used separately, they should probably be moved in a single `UpdateService` method.